src/AddThis.php

```diff
@@ -56,7 +56,8 @@ public function getServices() {
           '#type' => 'inline_template',
           '#template' => '<span class="addthis_service_icon icon_' . $serviceCode . '"></span> ' . $serviceName,
         );
-        $rows[$serviceCode] = $service;
+        //#options expects a string, not an array. Render the element so it becomes a string.
+        $rows[$serviceCode] = render($service);
```

Comment:
This will be moved as part of another issue. Had to call render() here because of the nature of the nested form options to avoid getting 500 warnings on every load of the admin page.